test/sagemaker_tests/pytorch/training/integration/sagemaker/test_pytorchddp.py

```diff
@@ -66,3 +68,37 @@ def test_pytorchddp_throughput_gpu(
         invoke_pytorch_estimator(
             ecr_image, sagemaker_regions, estimator_parameter, job_name=job_name
         )
+
+
+@pytest.mark.processor("gpu")
+@pytest.mark.model("N/A")
+@pytest.mark.multinode(2)
+@pytest.mark.integration("pytorchddp")
+@pytest.mark.parametrize(
+    "efa_instance_type", get_efa_test_instance_type(default=["ml.p4d.24xlarge"]), indirect=True
+)
+@pytest.mark.skip_cpu
+@pytest.mark.skip_py2_containers
+@pytest.mark.skip_trcomp_containers
+@pytest.mark.efa()
+@pytest.mark.skip_inductor_test
+def test_pytorchddp_throughput_gpu_mpi(
```

Comment:
Is this just a test addition that is useful to have, or is the mpi test explicitly needed for p5?

Reply:
Without sagemaker features, pytorchddp test needs to be run using mpi